app/api/v2/models/meetup_model.py

```diff
@@ -31,15 +31,29 @@ def save(self):
         self.cursor.execute(query)
         self.connection.commit()
 
+    def get_meetup_by_id(self, meetup_id, value):
+        '''Return a meetup given a meetup id'''
```

Comment:
Please use double quotes (`"`) for DocStrings - PEP8

Reply:
Alright, I will update in next PR.